meinberlin/apps/bplan/signals.py

```diff
@@ -28,6 +34,8 @@ def send_notification(sender, instance, created, **kwargs):
 @receiver(post_save, sender=Bplan)
 def send_update(sender, instance, update_fields, **kwargs):
     if not update_fields or (
-        "point" not in update_fields and "is_archived" not in update_fields
+        "point" not in update_fields
+        and "is_archived" not in update_fields
+        and "administrative_district" not in update_fields
```

Comment:
why is this email is sent? hard to understand with all these conditional checks.

Reply:
the `administrative_district` is not used in the task below. bit confusing.

Reply:
added some context and removed the field